packages/react-components/react-menu/stories/Menu/MenuDefault.stories.tsx

```diff
@@ -1,20 +1,84 @@
 import * as React from 'react';
```

Comment:
It seems like you updated this file to test things and the changes made here should be reverted.